modules/helfi_tpr_config/config/install/core.entity_view_display.tpr_service.tpr_service.teaser_search_result.yml

```diff
@@ -49,7 +49,7 @@ content:
           show: true
```

Comment:
Are these changes related to this PR?

Reply:
nope, config export just decided that those strings with empty space need quotes

Reply:
Non-related. drush export decided to add quoted to the string because of the space between the words.